keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -365,10 +367,27 @@ type JWTAuthenticator struct {
         CertificateAuthority string `json:"certificateAuthority,omitempty"`
 
         // audiences is the set of acceptable audiences the JWT must be issued to.
-        // At least one of the entries must match the "aud" claim in presented JWTs.
         // Same value as the --oidc-client-id flag (though this field supports an array).
         // Required to be non-empty.
         Audiences []string `json:"audiences,omitempty"`
+
+        // audienceMatchPolicy defines how the "audiences" field is used to match the "aud" claim in the presented JWT.
+        // Allowed values are "MatchAny" or "MatchAll" when multiple audiences are specified and
+        // empty when a single audience is specified.
+        //
+        // - MatchAny: the "aud" claim in the presented JWT must match at least one of the entries in the "audiences" field.
```

Comment:
MatchAny is the way I'd expect a server to be configured if it has multiple ways it can be addressed, and wants to accept a JWT for any of those URLs. There are clear reasons for this option.